presto-spi/src/main/java/com/facebook/presto/spi/ConnectorSession.java

```diff
@@ -53,4 +54,6 @@ default String getUser()
     <T> T getProperty(String name, Class<T> type);
 
     Optional<String> getSchema();
+
+    Optional<List<String>> getMetastoreHeaders();
```

Comment:
Can we have a `default` implementation of returning `Optional.empty()`

Reply:
sure makes sense :)